### PR TITLE
some extra medic tweaks

### PR DESCRIPTION
**Pull request by grinkgo**
## About The Pull Request

Just tuning some numbers for costs. 

numbers-wise, idfk really I just fiddled around with them so I'm open to feedback. 

## Why It's Good For The Game

its fine for medics to get some good chems but it should be at the cost of p much anything else they want in their loadout.

## Changelog
:cl:
add: Added a 60u syringe case of chems in medic vendor.
tweak: Tweaked costs of items in medic vendor.
tweak: Contents of medkit pouch changed.
tweak: Tweezers can fit in more pouches and belts.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Losenis** [comment]
Why should the 120u cases be more expensive than 2 60u cases? It turns into a noob trap with meraderm for example, where a corpsman should buy a 60u mera and 60u derma cases then mix them since a 120u meraderm case would cost 4 additional wasted points in total.

I firmly believe a 120u case should be a little cheaper than two 60u cases due to making you buy a far bigger quantity upfront that you might not even need and could lock you out of other purchases through those points that you never get more of.

Another trap I'm seeing is the 60u hypospray of a reagent that costs 3 more points than a case, even though an empty hypospray costs only 2 points. One point might seem like nothing, but it can be absolutely critical when it's the difference between getting another good purchase or not. Also no changes to the 120u empty hypospray still costing 10 points?

These changes aren't really giving clear pros and cons, they're just point sinks for those who don't yet know how to minmax the shit out of the vendor. The only benefit I'm seeing in some of these new stats is saving a minute in bottle juggling as the corpsman combines the various 60u cases, _which there's more than enough time for given prep time is around 20 to 30 minutes._ As corpsman I can easily write off a very good few of these vendor options as useless at best if not outright detrimental at worst.

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -968,23 +968,27 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
...
-		/obj/item/storage/syringe_case/dermaline = list(CAT_MEDSUP, "syringe Case (120u Dermaline)", 16, "orange"),
-		/obj/item/reagent_containers/hypospray/advanced/dermaline = list(CAT_MEDSUP, "hypospray (60u dermaline)", 8, "orange"), //half the units of the derm case half the price
-		/obj/item/storage/syringe_case/meraderm = list(CAT_MEDSUP, "syringe Case (120u Meraderm)", 16, "orange"),
-		/obj/item/reagent_containers/hypospray/advanced/meraderm = list(CAT_MEDSUP, "hypospray (60u Meraderm)", 8, "orange"), //half the units of the meraderm case half the price
+		/obj/item/storage/pill_bottle/bicamera = list(CAT_MEDSUP, "BicaMera pills", 20, "orange"),
+		/obj/item/storage/pill_bottle/keloderm = list(CAT_MEDSUP, "KeloDerm pills", 20, "orange"),
+		/obj/item/storage/pill_bottle/paracetamol = list(CAT_MEDSUP, "Paracetamol pills", 5, "black"),
+		/obj/item/storage/pill_bottle/bicaridine = list(CAT_MEDSUP, "Bicaridine pills", 7, "black"),
+		/obj/item/storage/pill_bottle/kelotane = list(CAT_MEDSUP, "Kelotane pills", 7, "black"),
+		/obj/item/storage/pill_bottle/tramadol = list(CAT_MEDSUP, "Tramadol pills", 5, "orange"),
```
Comment (Pariah919) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:976]:
```suggestion
		/obj/item/storage/pill_bottle/tramadol = list(CAT_MEDSUP, "Tramadol pills", 5, "orange"),
```
why are these 5 to seven points when you're able to get them for free???

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 2]
```diff
@@ -968,23 +968,27 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
...
+		/obj/item/storage/pill_bottle/tricordrazine = list(CAT_MEDSUP, "Tricordrazine", 7, "black"),
+		/obj/item/storage/syringe_case/meralyne = list(CAT_MEDSUP, "syringe Case (120u Meralyne)", 22, "black"),
+		/obj/item/storage/syringe_case/meralyne/half = list(CAT_MEDSUP, "syringe Case (60u Meralyne)", 10, "black"), //half the units of the mera case half the price
```
Comment (Pariah919) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:979]:
```suggestion
		/obj/item/storage/syringe_case/meralyne/half = list(CAT_MEDSUP, "syringe Case (60u Meralyne)", 10, "black"), //half the units of the mera case half the price
```
why is it more efficent pointwise to buy 2 meralyne 60us than one 120?
Why is this even an option in the first place

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 3]
```diff
@@ -968,23 +968,27 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
...
+		/obj/item/reagent_containers/hypospray/advanced/meralyne = list(CAT_MEDSUP, "hypospray (60u Meralyne)", 13, "black"),
```
Comment (Pariah919) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:980]: not sure why this is here consdering medics can already buy hyposprays and have to flush them to unfill them
this is a noob trap

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 4]
```diff
@@ -968,23 +968,27 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
...
+		/obj/item/storage/syringe_case/dermaline = list(CAT_MEDSUP, "syringe Case (120u Dermaline)", 22, "orange"),
```
Comment (Pariah919) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:981]: ditto to meralyne

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 5]
```diff
@@ -968,23 +968,27 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
...
+		/obj/item/storage/syringe_case/dermaline/half = list(CAT_MEDSUP, "syringe Case (60u Dermaline)", 10, "orange"), //half the units of the derm case half the price
+		/obj/item/reagent_containers/hypospray/advanced/dermaline = list(CAT_MEDSUP, "hypospray (60u dermaline)", 13, "orange"),
+		/obj/item/storage/syringe_case/meraderm = list(CAT_MEDSUP, "syringe Case (120u Meraderm)", 24, "orange"),
+		/obj/item/storage/syringe_case/meraderm/half = list(CAT_MEDSUP, "syringe Case (60u Meraderm)", 12, "black"), //half the units of the meraderm case half the price
```
Comment (Pariah919) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:985]: ditto to the above

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 6]
```diff
@@ -968,23 +968,27 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
...
+		/obj/item/reagent_containers/hypospray/advanced/meraderm = list(CAT_MEDSUP, "hypospray (60u Meraderm)", 15, "orange"),
 		/obj/item/storage/syringe_case/ironsugar = list(CAT_MEDSUP, "syringe Case (120u Ironsugar)", 5, "black"),
 		/obj/item/reagent_containers/hypospray/advanced/ironsugar = list(CAT_MEDSUP, "hypospray (60u Ironsugar)", 3, "black"), //bit more than half of the ironsugar case
-		/obj/item/storage/syringe_case/tricordrazine = list(CAT_MEDSUP, "syringe Case (120u Tricordrazine)", 5, "black"),
-		/obj/item/reagent_containers/hypospray/advanced/tricordrazine = list(CAT_MEDSUP, "hypospray (60u Tricordrazine)", 3, "black"),
+		/obj/item/storage/syringe_case/tricordrazine = list(CAT_MEDSUP, "syringe Case (120u Tricordrazine)", 8, "black"),
```
Comment (Pariah919) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:989]: ditto to the 5-7 points for normal meds

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 1 above)
Comment (Pariah919) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:976]: make them like 1-2 points or something

**Review by Pariah919 — COMMENTED** [review]
opinion

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 1 above)
Comment (fernandos13) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:976]: just remove from the thing. its a needless bloat. add to the corpsman med machine. that one that we get in PoS thats exclusive to corps prep

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 2 above)
Comment (fernandos13) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:979]: pariah got a point here.
so.. other thign you raised the price of 60u of mer/derm/meraderm from 8 to 10
and removed the hypo. thats 2 nerfs in one... the cost of hypos is already 2 points.. so at least. keep the hypo?

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 3 above)
Comment (fernandos13) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:980]: also this costs 3 points and hypos cost 2 .. its.. WORST than getting a hypo and mera. and filling it

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 6 above)
Comment (fernandos13) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:989]: trico is even more wide spread than the other chems. due to inaprov pills having 30u in each pill.. you can even bake trico inside bottles with 1 inap pill and 2 dylo. that come in EVERY corpsman belt

**Review by fernandos13 — COMMENTED** [review]
also 
```

/obj/item/reagent_containers/hypospray/advanced = list(CAT_MEDSUP, "Advanced hypospray", 2, "black"),
/obj/item/reagent_containers/hypospray/advanced/big = list(CAT_MEDSUP, "Big hypospray", 10, "black"),
```

big hypo is not only TWICE the price of the regular hypo. (for holding twice the reagents but its 5 times.. the price.. FIVE.. no one is getting this

the medkit pouch is a CLEAR trap. its ok to have it there. altough i dont see why ANYONE would give up on 60u of meralyine 60u of derma. 1 peri plu and one qc plus injector. and also a synaps one for a pouch full of crap even marines can get.....

but at least make it not marked as "essential"
or remove 
1 brute kit for advanced hypo
1 burn kit For a peri plus
1 splint For a QC plus
1 hl analyzer (we already get TWO of those from essential med kit)  for synaps. (which people drop in prep but would at least keep in line with the other pouches)

**Comment by fernandos13** [comment]
also remove the "pistol holster from the webbings" its just.. not fit for corpsman we struggle with storage... and that wont even provide something worth of giving up on a regular webbing. and if possible get the surgery vest back in there. even if empty .....
i rather have 14 brute kits. than 6 common medical items that i can store in the backpack

**Comment by fernandos13** [comment]
maybe if you want push the use of medkit pouches

make so it can hold:
tweezers
defib
medvac
rollerbed
AND THE FUCKING SYRINGE CASE.. that even a common medkit can. and lots of them even come with

**Comment by fernandos13** [comment]
also try to keep in mind. corpsman get 45 points to use up in the vendor. and lots of the advanced chems can only be gathered here. since capture gameplay was pushed out.. and neuraline is DEFINELTY a exclusive chem here.

so balance with that in mind. cause at the moment people will want to spend the most they can in T2 meds
like
meralyine
dermaline
neuraline (definetly my personal go to.. since its so unobtainable otherwise)
so before one could get 
a meraderm kit 16 (2x8) or just (1x16)
and 2 neuraline inject 28 (2x 14)

and would be left with one point. and then proceed to medbay to hack the vendor for 2 hypos... that i would empty and fill with custom mixes
not to mention to get 60u of dylo and make a cheaper imialk mix
and then getting bic kelo tram. bottles to refill my hypos
i would empty the oxy hypo for a tram inap mix
and i would remove the oxy from my ADV hypo for tram too

atm after the last merge
i added
rushing to medbay to steal the surgical vest in MO locker. since surgery already got a medical tray for the MOs

this changes are progressively getting prep time to be longer (aka having to buy meraderm 60u and Hypo. and using it to fill the hypo) its a small thing. but we got so many of that in corpsman prep that sums up for a larger prep time

**Comment by grinkgo** [comment]
@Losenis Hey thanks for the feedback! Here are some of my thoughts:

> I firmly believe a 120u case should be a little cheaper than two 60u cases due to making you buy a far bigger quantity upfront that you might not even need and could lock you out of other purchases through those points that you never get more of.
> 
- good point on the 120  vs the 60. I suppose it should make more sense that the "two for one" should be slightly cheaper than two individually, not more expensive. In which case I'm thinking to swap their costs or some similar adjustment. 

> Why should the 120u cases be more expensive than 2 60u cases? It turns into a noob trap with meraderm for example, where a corpsman should buy a 60u mera and 60u derma cases then mix them since a 120u meraderm case would cost 4 additional wasted points in total.
> 
- Hm, my thought going into it was that if you wanted both mera AND derm it should be more expensive, but yea the addition of the 60u changes that so my numbers here don't make sense. Part of this should be fixed then I think if I make the combined cost of 60 of each equal to the 120, and then make the 60u of meraderm more expensive than the 60u of either individually.

> Another trap I'm seeing is the 60u hypospray of a reagent that costs 3 more points than a case, even though an empty hypospray costs only 2 points. One point might seem like nothing, but it can be absolutely critical when it's the difference between getting another good purchase or not. Also no changes to the 120u empty hypospray still costing 10 points?
> 
- So the hypospray costing 3 more than a case is accounting for the cost of the hypo itself (2) and a 1 point "convenience" fee to just have a premade hypo with the chem in it. Buying the individual bottles is more for like, putting them into bigger mixtures. If you wanna buy a 60 and an empty hypo and fill it yourself, that's your prerogative I suppose and you'll save a point in doing so.

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 1 above)
Comment (grinkgo) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:976]: Mhm, I can def make the basic pills cheaper.

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 2 above)
Comment (grinkgo) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:979]: Addressed in my reply to Losenis below

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 3 above)
Comment (grinkgo) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:980]: You're referring to the meralyne hypospray?

**Comment by grinkgo** [comment]
> also
> 
> ```
> 
> /obj/item/reagent_containers/hypospray/advanced = list(CAT_MEDSUP, "Advanced hypospray", 2, "black"),
> /obj/item/reagent_containers/hypospray/advanced/big = list(CAT_MEDSUP, "Big hypospray", 10, "black"),
> ```
> 
> big hypo is not only TWICE the price of the regular hypo. (for holding twice the reagents but its 5 times.. the price.. FIVE.. no one is getting this
> 
> the medkit pouch is a CLEAR trap. its ok to have it there. altough i dont see why ANYONE would give up on 60u of meralyine 60u of derma. 1 peri plu and one qc plus injector. and also a synaps one for a pouch full of crap even marines can get.....
> 
> but at least make it not marked as "essential"
> or remove
> 1 brute kit for advanced hypo
> 1 burn kit For a peri plus
> 1 splint For a QC plus
> 1 hl analyzer (we already get TWO of those from essential med kit) for synaps. (which people drop in prep but would at least keep in line with the other pouches)

1. Yeah I'm def open to change up the contents of the medkit pouch. and also what you mentioned in the following comment about letting it hold some stuff like tweezers and the syringe cases.
2. Cost of big hypo is also subject to change. I think then **if** say the 60u hypo is 2 points it's more appropriate for the 120u to be around 5-ish.
3. I can just buff the corpsman webbing but idk MJP's the one waging the storage war. That part isn't really my territory. I personally am never returning the surgery webbing to medics.
4. Regarding the price of the premade hypo vs a bottle and an empty hypo, for the most part I intend to keep this general idea. The cost here is 1 vendor point vs the time you spend filling a hypo manually.

**Comment by Losenis** [comment]
> and a 1 point "convenience" fee to just have a premade hypo with the chem in it
Please don't add convenience fees, they sound good on paper but in practice it's just punishing for not minmaxing since these points are extremely limited. My corpsman loadout already involves 28 points spent on two neuraline injectors, 'convenience' really isn't a thing for these kinds of vendors where you're just never getting any more points than you receive upon joining the round.

Bigger purchases and premixes should be cheaper because it'd be an investment towards a specific goal, with the meraderm premix you wouldn't have the luxury of giving only mera or derm to a guy who only has one damage type (unless a guy autistically pills them in the chem master, but that's so specific and tedious that I wouldn't even consider that in the argument. Power to him I guess. It'd be balancer around the fact other corpsmen would want to frag him for taking up even more chem time.)

Even if one doesn't use all the points in prep, that's still points left as safety net for the corpsman to come back and use to restock later on. That cements how things like convenience fees are just noob traps. Going back to the 120u meraderm example, if a loadout is based around only 60u of it, it'd be wasteful to get 120u. It'd be more point efficient to get 120u, but it could very well lock you out of buying something else. Getting 60u of meraderm also locks you to a 1:1 ratio of mera and derma. Assuming a change where 120u MeraDerm is a bit cheaper than 60u Mera + 60u Derma, it'd be more cost efficient to buy the mix if you wanted both, but if you only needed one for some reason, you'd be rewarded for  buying a case that has only one and not the other, while gambling on the idea that if you do buy a case of the other too, you just wasted a few points by not getting the mix. It's clear pros and cons even for niche instances.

**Comment by grinkgo** [comment]
> > and a 1 point "convenience" fee to just have a premade hypo with the chem in it
> 
> Please don't add convenience fees, they sound good on paper but in practice it's just punishing for not minmaxing since these points are extremely limited. My corpsman loadout already involves 28 points spent on two neuraline injectors, 'convenience' really isn't a thing for these kinds of vendors where you're just never getting any more points than you receive upon joining the round.
> 
> Bigger purchases and premixes should be cheaper because it'd be an investment towards a specific goal, with the meraderm premix you wouldn't have the luxury of giving only mera or derm to a guy who only has one damage type (unless a guy autistically pills them in the chem master, but that's so specific and tedious that I wouldn't even consider that in the argument. Power to him I guess. It'd be balancer around the fact other corpsmen would want to frag him for taking up even more chem time.)
> 
> Even if one doesn't use all the points in prep, that's still points left as safety net for the corpsman to come back and use to restock later on. That cements how things like convenience fees are just noob traps. Going back to the 120u meraderm example, if a loadout is based around only 60u of it, it'd be wasteful to get 120u. It'd be more point efficient to get 120u, but it could very well lock you out of buying something else. Getting 60u of meraderm also locks you to a 1:1 ratio of mera and derma. Assuming a change where 120u MeraDerm is a bit cheaper than 60u Mera + 60u Derma, it'd be more cost efficient to buy the mix if you wanted both, but if you only needed one for some reason, you'd be rewarded for buying a case that has only one and not the other, while gambling on the idea that if you do buy a case of the other too, you just wasted a few points by not getting the mix. It's clear pros and cons even for niche instances.

Awesome feedback, and on that note I can agree to remove the convenience fee. Will also take some of this into mind as a touch up on costs.

**Comment by fernandos13** [comment]
ALSO i reminded the name
terragov medic vendor

we could use that for some other stuff that doesnt require points. 

like brute kits. 
burn kits
empty pouches

**Comment by fernandos13** [comment]
> * So the hypospray costing 3 more than a case is accounting for the cost of the hypo itself (2) and a 1 point "convenience" fee to just have a premade hypo with the chem in it. Buying the individual bottles is more for like, putting them into bigger mixtures. If you wanna buy a 60 and an empty hypo and fill it yourself, that's your prerogative I suppose and you'll save a point in doing so.

thats just counter QoL... paying convenienve fee.... is a awlful idea.. for corpsman prep. since its already one of the longest preps we got

**Comment by fernandos13** [comment]
`Regarding the price of the premade hypo vs a bottle and an empty hypo, for the most part I intend to keep this general idea. The cost here is 1 vendor point vs the time you spend filling a hypo manually.`

id take the time any day. ... im already missing half the drops that goes down in 12:24 instead of 12:25 anyway
plus i waste 28 in neura injectors since they are so hard to get
and if i can leave some points to restock later i'll cause meraderm is just in pratice impossible to get anywhere else

but i dont feel like i got enough point to even build my loadout. 
also on the 120 vs 60 discussion

some people might "buying in bulk should have discount" as a thing but i disagree
the reason i added 60u bottles to start with was literally to make so i could buy 60u now. and fit in my loadout . then when it got empty restock with another 60u. instead of being forced to carry 60u that shorter ops would not use... or even the flexibility of choose if i want 2 injectors of merader. and 1 of each for more precise dosing ((abd both sides have its advantages. if you get it mixed.. you have no option but to waste BOTH..  on the marine. at 1:1 rate. while if ou get 60u of each. you can now selected which medicine you wanna use. but in the end. you might use a whole mera hypo and be with only half of a derma used. while in the meraderm case. if will usually get 2 hypos of the mix resulting in a more equal "outage" of both
aand the space. those items takes are very often overlooked. 
one of the biggest reasons i rather take the hypo (aside from the quick injectrion. is that carrying a hypo that is the same size of a bottle. is less space than A BOTTLE AND A NEEDLE..

File: code/game/objects/items/storage/belt.dm [hunk 7]
```diff
@@ -173,13 +176,13 @@
...
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/combat(src)
+	new /obj/item/storage/pill_bottle/dexalin(src)
+	new /obj/item/storage/pill_bottle/quickclot
+	new /obj/item/reagent_containers/hypospray/autoinjector/combat_advanced(src)
 	new /obj/item/storage/syringe_case/combat(src)
 	new /obj/item/storage/syringe_case/combat(src)
 	new /obj/item/storage/syringe_case/combat(src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/quickclot(src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/quickclotplus(src)
```
[the remark this answers is not in the document]
Comment (grinkgo) [reply code/game/objects/items/storage/belt.dm:185]: having two versions of essentially the same thing is kinda retarded ngl. But I can make them identical. I would have to replace all instances of them tho which is pain

**Pull request closed without merging**